var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -94,6 +94,8 @@ def install(self, spec, prefix):
         config = Executable('./config')
         config('--prefix=%s' % prefix,
                '--openssldir=%s' % join_path(prefix, 'etc', 'openssl'),
+               '-I{0}'.format(self.spec['zlib'].prefix.include),
```

Comment:
Both should be in compiler wrappers...can u not set ‘—zlibdir=‘?

Reply:
I tried the default `--with-zlib=...` but that weird self-made script does not have those... :)

Reply:
oh wait, I see what you mean. `--zlibdir=` ... have to try!

Reply:
I don’t know if it exists, but it follows: --openssldir

Reply:
nope, that one is also not supported:
`gcc-5: error: unrecognized command line option '--zlibdir=/usr/local/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/zlib-1.2.11-swly52at4ohpwlzmkiwejxybzrij5w5g`